charts/fulcio/Chart.yaml

```diff
@@ -5,8 +5,8 @@ description: |
 
 type: application
 
-version: 0.5.2
-appVersion: 0.6.0
+version: 0.9.9
```

Comment:
Happy to change if there's a preference, I just thought we should signify this is approaching 1.0

Reply:
It is a bit risky in case we need to update the chart with a new release candidate in case of bugs before 1.0

Reply:
I was thinking we could just add .9 again for each rc. Happy to revert this back though and just increment the minor version until 1.0, no strong preference!

Reply:
It is okay with me, just wanted to know how we'll deal with more RCS.